src/extractor/MappingExtractor.ts

```diff
@@ -76,7 +77,7 @@ export class MappingExtractor {
   static extractRules(element: ProcessableElementDefinition, mappings: ExportableMapping[]) {
     element.mapping?.forEach((mapping, i) => {
       // Mappings are created at SD, so should always find a match at this point
-      const matchingMapping = mappings.find(m => m.name === mapping.identity);
+      const matchingMapping = mappings.find(m => m.id === mapping.identity);
```

Comment:
Why do we use the `id` here instead of the `name`? I thought the `name` was the one that would be unique, but the `id` may not be?

Reply:
I think it is because the `name` is a FSH-only construct.  It does not have any representation in FHIR, so we can't compare it against a value from the FHIR mapping.  So the correct thing to compare the FHIR `mapping.identity` to is the FSH Mapping `id`.

While `name` is unique across the whole FSH Tank, `id` _should_ be unique across the `StructureDefinition`, and that's all that matters in this case since FHIR `mapping`s are scoped within the SD only.

Reply:
What Chris said!